src/kernel/diffusive/diffusive.f90

```diff
@@ -216,6 +217,13 @@ subroutine diffnw(timestep_ar_g, nts_ql_g, nts_ub_g, nts_db_g, ntss_ev_g, nts_qt
     double precision, dimension(:,:), allocatable :: used_lfrac    
     double precision, dimension(:,:,:), allocatable :: temp_q_ev_g
     double precision, dimension(:,:,:), allocatable :: temp_elv_ev_g
+    
+    open(unit=11, file="./output/cn-mod results at timesteps.txt")
```

Comment:
Are any of these file units required output?  If not, they need to be removed before this can go operational

Reply:
Removed.